Private/Invoke-Command.ps1

```diff
@@ -0,0 +1,19 @@
+function Invoke-Command {
+    [CmdletBinding()]
+    param (
+        [Parameter(mandatory=$true)]
+        [string]
+        $Command
+    )
+
+    try {
+        $result = [CommandResult]::new()
+        $output = Invoke-Expression "& $Command" 2> $null
```

Comment:
It would be nice to print out the actual command being used when debugging is enabled.

Reply:
I am interested in the error messages when a command fails. Would point to the right direction to solve the issue though, instead of being swallowed by `$null`. :)

Reply:
Now stderr gets redirected to stdout so at least you get everything in the output which is the best we can do given powershell's current limitations.